tower_cli/resources/ad_hoc.py

```diff
@@ -85,11 +79,7 @@ class Resource(models.ExeResource):
                   help='If provided with --monitor, this attempt'
                        ' will time out after the given number of seconds. '
                        'Does nothing if --monitor is not sent.')
-    @click.option('--become', required=False, is_flag=True,
-                  help='If used, privilege escalation will be enabled for '
-                       'this command.')
```

Comment:
It looks like maybe we aliased `become_enabled` to `become`? I'd say should go in the changelog, but I have no objections.